Finjector.Web/Controllers/ChartsController.cs

```diff
@@ -50,6 +50,10 @@ public async Task<IActionResult> GetChart(int id)
                return NotFound();
         }
         rtValue.CanEdit = await _userService.VerifyChartAccess(id, iamId, Role.Codes.Edit);
+        if(!rtValue.CanEdit)
+        {
+            rtValue.FolderId = 0;
```

Comment:
i think i'd rather it be handled in the UX.  Having the API clear out the legit folderId doesn't seem quite right

Reply:
Not sure how to do that.